assets/js/modules/search-console/components/dashboard/DashboardClicksWidget.js

```diff
@@ -52,10 +52,11 @@ function DashboardClicksWidget() {
 		const isDomainProperty = select( STORE_NAME ).isDomainProperty();
 		const referenceSiteURL = untrailingslashit( select( CORE_SITE ).getReferenceSiteURL() );
 
+		const dateRange = select( CORE_USER ).getDateRangeDates( { compare: true } );
 		const args = {
 			dimensions: 'date',
-			compareDateRanges: true,
-			dateRange: select( CORE_USER ).getDateRange(),
+			startDate: dateRange.compareStartDate,
+			endDate: dateRange.compareEndDate,
```

Comment:
The IB here is actually slightly incorrect, I've missed that before. My comment https://github.com/google/site-kit-wp/issues/2237#issuecomment-728529776 has the intended behavior though: The `compare*Date` values represent a secondary date range (in addition to `startDate` and `endDate`) that occurs right before that main date range. However the Search Console API doesn't support passing two date ranges, so what we'll need to do is pass it as follows to encompass the entirety of the two date ranges:
```suggestion
			// Combine both date ranges into one single date range.
			startDate: dateRange.compareStartDate,
			endDate: dateRange.endDate,
```